demo/iaa-qpl-demo/Dockerfile

```diff
@@ -12,39 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:20.04 AS builder
+FROM ubuntu:22.04 AS builder
 
-RUN apt update && DEBIAN_FRONTEND=noninteractive TZ="Etc/UTC" \
-    apt install -y --no-install-recommends \
-    g++ clang nasm cmake make git ca-certificates uuid-dev \
-    gcc autoconf automake libtool pkg-config libjson-c-dev curl zlib1g-dev
+RUN apt update && apt install -y --no-install-recommends \
```

Comment:
Why you're removing `DEBIAN_FRONTEND=noninteractive`?

Also, please run the Dockerfile through `hadolint`.

PS. Feel free to use `.hadolint.yaml` config file like this:
```
ignored:
# use Docker "--no-cache" to get security updates instead
# of tracking and constantly updating pinned package versions:
- DL3008
```